store/query.go

```diff
@@ -30,3 +30,12 @@ type Sort struct {
 	AttrName  string
 	Ascending bool
 }
+
+type ListQuery struct {
```

Comment:
confusing name, did not know what it is until looked at implementation
probably something like Filter, QueryFilter is closer to what it does